analyses/snv-callers/README.md

```diff
@@ -0,0 +1,211 @@
+# SNV caller comparison analysis
+
+This analysis evaluates [MAF files](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)
+from different SNV callers and compares their output.
+The GDC has [good documentation on the fields](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) contained in a standard MAF file.
+
+**Table of Contents**
+* [Individual caller evaluation](#individual-caller-evaluation)  
+  * [Base change analysis](#base-change-analysis)  
+  * [Variant allele fraction calculation](#variant-allele-fraction-calculation)  
+  * [Genomic regional analyses](#genomic-regional-analyses)  
+  * [Tumor mutation burden](#tumor-mutation-burden-calculation)
+  * [COSMIC mutation overlap](#cosmic-mutation-overlap)  
+* Comparison analysis - in development
+ * [Mutation IDs](#mutation-ids)
+* [Overall file structure](#overall-file-structure)
+* [Summary of functions](#summary-of-special-functions)
+## Individual Caller Evaluation
+
+The first step in this analysis is an individual evaluation of each MAF from
+each caller.
+This analysis prints out a report that is further subdivided by the WGS and WXS
+samples.
+
+Overall reports for each caller and strategy can be found:
+* `results/<caller_name>/<caller_name>_wgs_report.html`
+* `results/<caller_name>/<caller_name>_ws_report.html`
+
+### Base change analysis
+
+To evaluate base changes I summarized standard MAF fields as two new variables:
+The `base_change` variable that indicates the exact change in bases from
+concatenating `Reference_Allele`, `>`, and `Allele`.
```

Comment:
I agree with your 'dumb' comment, but because `>` is "standard" notation, I'm gonna stick with it. 
Yeah, looking at the flanking bases is also is probably a good thing to do and I think it will be something I address in tandem with looking at the mutational signatures at a later step. But I think that will be after we decide which mutations we decide are "true" (which is the first goal here perhaps I should have made that more clear).

Reply:
Maybe change it for the plots? (More trouble than it is worth for now, but I'm leading the revolution when we get to publication-ready plots.)